README.md

```diff
@@ -129,24 +150,30 @@ jupyter kernelspec uninstall pymbe
 The JupyterLab main menu will now show a new Python kernel called "pyMBE"
 that uses the virtual environment.
 
-### Use pyMBE in your simulation scripts
+### Run simulation scripts
+
+You can run the branched polyampholyte sample with the following commands:
 
 ```sh
 source pymbe/bin/activate
-python3 samples/peptide.py
+python3 samples/branched_polyampholyte.py --pH 6
+python3 samples/analyze_time_series.py --data_folder samples/time_series/branched_polyampholyte
+python3 samples/plot_branched_polyampholyte.py
 deactivate
 ```
 
-### Run the tutorial of pyMBE
+### Run tutorials
 
-You can run the interactive tutorial of pyMBE with the command:
+You can run the interactive tutorials with the following commands:
 
 ```sh
 source pymbe/bin/activate
-jupyter-lab tutorials/pyMBE_tutorial.ipynb
+jupyter-lab
```

Comment:
The old syntax would start the Jupyter server inside the `tutorials` folder and open `pyMBE_tutorial.ipynb`. While it appears to be convenient, the notebook was actually unable to display the diagrams, because relative paths of the form `../figs/image.png` are blocked by web servers for security reasons (otherwise a malicious actor would be able to traverse the entire file system of the machine the notebook is running on). Starting Jupyter in the top-level directory makes the `figs/` folder walkable by the web server.